.pre-commit-config.yaml

```diff
@@ -0,0 +1,5 @@
+repos:
+-   repo: https://gitlab.com/pycqa/flake8
+    rev: 3.7.9
+    hooks:
+    - id: flake8
```

Comment:
Do we want to add "black" as a pre-commit hook?  Could help to have that for standard formatting.

Reply:
Okay, black seems to create some conflicts with the flake8 configuration(!). I will put it in as a suggestion but with a caveat.